packages/node-experimental/src/integrations/http.ts

```diff
@@ -161,7 +158,7 @@ export class Http implements Integration {
       return;
     }
 
-    const data = getRequestSpanData(span);
+    const data = _INTERNAL.getRequestSpanData(span);
```

Comment:
m: This `_INTERNAL` thing seems very smelly to me. Can we not hoist this into utils or smth?

Reply:
yeah so this should _definitely_ go away, I just want to avoid exposing things publicly that we need _right now_ because then we can't remove them easily later :) once the semantic attributes stuff is merged and agreed upon this can/should just be public API!